tests/python/frontend/onnx/test_forward.py

```diff
@@ -2374,17 +2374,11 @@ def test_pooling():
                        auto_pad='SAME_UPPER')
 
 
-def verify_mod(x_shape, y_shape, fmod, dtype='float32'):
+def verify_mod(x_shape, y_shape, fmod, out_shape, dtype='float32'):
     x_np = np.random.uniform(size=x_shape).astype(dtype)
     y_np = np.random.uniform(size=y_shape).astype(dtype)
```

Comment:
These tests are only passing because `np.random.uniform` is always positive. If you use negative numbers, the behavior of `mod` changes based on the value of `fmod`. We should use a different random input generator than uniform anyway since values between 0 and 1 dont make sense for mod testing anyway.